ui/src/map/layers/operational-point/operational-points-layer-utils.ts

```diff
@@ -20,6 +20,8 @@ export enum OperationalPointFeatureSize {
     Small,
 }
 
+export type OperationalPointFeatureMode = 'DELETED' | 'HIGHLIGHTED' | 'SELECTED' | 'REGULAR';
```

Comment:
Tämä tuli uutena korvaamaan `isHighlightedOrSelected`-booleanin. Myös esikatselunäkymän poistetut toiminnalliset pisteet piirretään käyttäen näitä utileja